cammac60/macrae-software
# Basic setup

## What does this PR do? 

- Adds basic file structure.
- Adds basic imports to all files. 
- Removes unneeded boiler plate files. 
- Sets up test suite.

----------

### How to manually test: 

- `git fetch`
- `git checkout basic-setup`
- Ensure that everything looks correct. 

---------

### Tickets: 

- Resolves #2 
- Resolves #4 
- Resolves #5 

----------

### Notes: 

 None
